src/org/thoughtcrime/securesms/WebViewActivity.java

```diff
@@ -34,6 +37,8 @@ public class WebViewActivity extends PassphraseRequiredActionBarActivity
                                           WebView.FindListener
 {
   private static final String TAG = WebViewActivity.class.getSimpleName();
+  protected static final int REQUEST_CODE_FILE_PICKER = 51426;
+  protected ValueCallback<Uri[]> mFileUploadCallbackSecond;
```

Comment:
filePathCallback would be a better name, the "second" part comes from another usage in AdvancedWebview

Reply:
definitely, yes